tool/tsh/common/recording_export.go

```diff
@@ -54,43 +56,56 @@ func onExportRecording(cf *CLIConf) error {
 	authClient := proxyClient.CurrentCluster()
 	defer authClient.Close()
 
-	fname := cf.OutFile
-	if fname == "" {
-		fname = cf.SessionID + ".avi"
+	filenamePrefix := cf.SessionID
+	if cf.OutFile != "" {
+		// trim the extension if it was provided (we'll add it later on)
+		filenamePrefix = strings.TrimSuffix(
+			strings.TrimSuffix(cf.OutFile, ".avi"), ".AVI")
 	}
 
-	frames, err := writeMovie(cf.Context, authClient, session.ID(cf.SessionID), fname)
-	// there may be a partial file, even if we encountered an error,
-	// so we indicate to the user when we wrote something
-	if frames > 0 {
-		fmt.Printf("wrote recording to %v\n", fname)
+	_, err = writeMovie(cf.Context, authClient, session.ID(cf.SessionID), filenamePrefix, fmt.Printf)
```

Comment:
Either here or within the function we should write out that we are starting the writing of recordings.  

As it takes time to keep writing more files I would append the "wrote uuid.avi" with 
```bash
Starting recording export
wrote uuid.avi
Continuing export to next file.
...
```
Otherwise you could think there is a error since it output that it wrote the file but doesn't tell you it's continuing.